casepro/statistics/models.py

```diff
@@ -55,6 +56,8 @@ def encode_scope(*args):
             return 'partner:%d' % args[0].pk
         elif types == [Org, User]:
             return 'org:%d:user:%d' % (args[0].pk, args[1].pk)
+        elif types == [Case]:
+            return 'case:%d' % (args[0].pk,)
```

Comment:
The ticket says `Add case count to Partners + Users`, I guess we don't need `[Case, User]`, maybe just `[User]` (and maybe change this one to `[Partner]`?). Since a case can be assigned to both a partner, as well as a user within that partner, I think we want to track both.

Having us aggregate over the case ID will just give us a total of 1 for each case, which will drop down to 0 when the case is closed? (I'm assuming that we want to track the number of open cases per partner and per user)